routes/potRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const potController = require("../controllers/potController");
+
+const router = express.Router();
+
+router.post("", potController.potsListFilterData);
```

Comment:
```suggestion
router.get("", potController.potsListFilterData);
```
데이터를 받아오는 것인데 post는 적합하지 않습니다.